src/test/scala/is/hail/methods/LinearRegressionBurdenSuite.scala

```diff
@@ -0,0 +1,173 @@
+package is.hail.methods
+
+import is.hail.SparkSuite
+import is.hail.expr.TDouble
+import is.hail.utils._
+import org.apache.spark.sql.Row
+import org.testng.annotations.Test
+
+class LinearRegressionBurdenSuite extends SparkSuite {
+
+  @Test def testWeightedSum() {
+    /*
+          123
+    Gene1 --
+    Gene2  -
+    Gene3 ---
+
+    Variant A       B       C       D       E       F       Weight
+    1       0.0     1.0     0.0     0.0     0.0     1.0     1.0
+    2         .     2.0       .     2.0     0.0     0.0     2.0
+    3       0.0       .     1.0     1.0     1.0       .     3.0
+
+    Sample  Cov1    Cov2    Pheno   Gene1   Gene2   Gene3
+    A       0.0     -1.0    1.0     0.0     0.0     0.0
+    B       2.0     3.0     1.0     5.0     4.0     5.0
+    C       1.0     5.0     2.0     0.0     0.0     3.0
+    D       -2.0    0.0     2.0     4.0     4.0     7.0
+    E       -2.0    -4.0    2.0     0.0     0.0     3.0
+    F       4.0     3.0     2.0     1.0     0.0     1.0
```

Comment:
This data is readily accessible from the files, including it here makes it easy to have the two sources go out of sync. I'm inclined to remove it.